website/docs/reference/resource-configs/dbt_valid_to_current.md

```diff
@@ -0,0 +1,96 @@
+---
+resource_types: [snapshots]
+description: "Snapshot dbt_valid_to_current custom date"
+datatype: "{<dictionary>}"
+default_value: {"dbt_valid_from": "dbt_valid_from", "dbt_valid_to": "dbt_valid_to", "dbt_scd_id": "dbt_scd_id", "dbt_updated_at": "dbt_updated_at"}
+id: "dbt_valid_to_current"
+---
+
+Available in 1.9 or with [Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) dbt Cloud.
+
+<File name='snapshots/schema.yml'>
+
+```yaml
+snapshots:
+  my_project:
+    +dbt_valid_to_current: "to_date('9999-12-31')"
+
+```
+
+</File>
+
+<File name='snapshots/<filename>.sql'>
+
+```sql
+{{
+    config(
+        unique_key='id',
+        strategy='timestamp',
+        updated_at='updated_at',
+        dbt_valid_to_current='to_date('9999-12-31')'
+    )
+}}
+```
+
+</File>
+
+<File name='dbt_project.yml'>
+
+```yml
+snapshots:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +dbt_valid_to_current: "to_date('9999-12-31')"
+```
+
+</File>
+
+## Description
+
+Use the `dbt_valid_to_current` config to set a custom future date for `dbt_valid_to` in new snapshot columns. When set, dbt will use this specified value instead of `NULL` for `dbt_valid_to` in the snapshot table.
+
+This simplifies makes it easier to assign date, work in a join, and range-based filtering with an end date.
```

Comment:
This sentence sounds funny, but I don't know how to re-work it. But I bet you'll have some ideas 🧠